recipes/mike/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "mike" %}
+{% set version = "0.5.3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mike-{{ version }}.tar.gz
+  sha256: eef9c6b146f20a18ff7e0512838fd57d4dc2f389d1c012fa2285c9642794aa85
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - mike = mike.driver:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - flake8
+    - pip
+    - pypandoc
+    - python
+  run:
+    - jinja2
+    - mkdocs >=1.0
+    - packaging
+    - python
+    - ruamel.yaml
+
+test:
+  imports:
+    - mike
+    - mike.templates
+  commands:
+    - pip check
+    - mike --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/jimporter/mike
+  summary: Manage multiple versions of your MkDocs-powered documentation
+  license: BSD-3-Clause
+  license_file: LICENSE
```

Comment:
Please add a comment:

```yaml
# License file manually packaged. See https://github.com/jimporter/mike/pull/23
license_file: LICENSE
...
```

I've opened a PR to fix the need to manually package the license file (https://github.com/jimporter/mike/pull/23). The comment as a reminder to check the status of this for future updates in the feedstock. Once it's in the source distribution, the local copy can be removed.